_storage/authentication.md

```diff
@@ -0,0 +1,31 @@
+---
+layout: storage
+permalink: /:collection/:path.html
+---
+# Authentication and storage
+
+Blockstack authentication is a bearer token-based authentication system. From an app user's perspective, login similar to third-party authentication techniques that they're familiar with. For an app developer, the flow is unlike the typical client-server flow of centralized sign-in services such as OAuth. With Blockstack the authentication flow happens entirely client-side.
```

Comment:
I think this is sentence is missing a word?

Reply:
The last one...it is a bit "spoken" structured. Corrected.